notebooks/hybrid_rag_pipeline_with_breakpoints.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    from haystack.components.builders.answer_builder import AnswerBuilder
</pre><p>the two lines with builder imports can be merged into one</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>